### PR TITLE
clarify api access

### PR DESCRIPTION
**Pull request by MarkWoulfeNeo4j**

### CODE REVIEW

**Comment by MarkWoulfeNeo4j** [comment]
[Clarify API Admonition](https://trello.com/c/82Vvsb9C/274-clarify-api-admonition)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-aura-300.surge.sh

File: modules/ROOT/pages/platform/api/authentication.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ The Aura API uses OAuth 2.0 for API authentication.
 
 [NOTE]
 ====
-Users with Free and Professional instances can create API credentials as long as they have entered billing information or are a member of a marketplace tenant.
+In addition to Enterprise users, users with Free and Professional instances can also create API credentials as long as they have entered billing information or are a member of a marketplace tenant.
```
Comment (nvitucci) [inline modules/ROOT/pages/platform/api/authentication.adoc:11]:
```suggestion
Enterprise users and users with Free and Professional instances can create API credentials as long as they have entered billing information or are a member of a marketplace tenant.
```

Just not a fan of "users, users".

Comment (MarkWoulfeNeo4j) [reply modules/ROOT/pages/platform/api/authentication.adoc:11]: While I understand your position, the re-write lacks the clarity that it is just the free and pro instance users that need the billing information, not the enterprise users.

Comment (nvitucci) [reply modules/ROOT/pages/platform/api/authentication.adoc:11]: Ah, I didn't get that difference. In this case I would rephrase:

```suggestion
Enteprise users can always create API credentials.
Users of Free and Professional instances can create API credentials as long as they have entered billing information or are a member of a marketplace tenant.
```

**Review by nvitucci — APPROVED** [review]
